apps/meteor/app/lib/server/startup/rateLimiter.js

```diff
@@ -47,9 +47,9 @@ DDPRateLimiter._increment = function (input) {
 
 // Need to override the meteor's code duo to a problem with the callback reply
 // being shared among all matchs
-RateLimiter.prototype.check = function (input) {
+RateLimiter.prototype.check = async function (input, sess) {
```

Comment:
I guess we should move this patch to the main code, since we have the whole code now.

Reply:
The patch is actually ported to inside the package, but kept this one in there just in case 🤔 will try to remove